Author: Louis Lagendijk <louis.lagendijk@gmail.com>
Date:   Tue Jul 31 01:31:39 2012 +0200
Origin: https://lists.fedoraproject.org/pipermail/scm-commits/2012-July/835925.html

disable evolution shell

Index: roger-router-1.8.2/configure.ac
===================================================================
--- roger-router-1.8.2.orig/configure.ac	2013-12-18 12:09:36.000000000 +0000
+++ roger-router-1.8.2/configure.ac	2014-01-14 12:56:14.386380138 +0000
@@ -434,7 +434,6 @@
 	[$PKG_CONFIG --atleast-version=3.5.4 libebook-1.2 && with_ebook_registry=yes
 	if test "x$with_ebook_registry" = xyes; then
 		AC_DEFINE([HAVE_EBOOK_SOURCE_REGISTRY], 1, [Have libebook >= 3.5.4])
-		PKG_CHECK_MODULES([EVO_SHELL], evolution-shell-3.0 >= 3.5.4,, AC_MSG_ERROR([libebook >= 3.5.4 requires evolution-shell]))
 	fi
 ])
 AM_CONDITIONAL(HAVE_EBOOK_SOURCE_REGISTRY, test "x$with_ebook_registry" = "xyes")
