Subject: Fix manpage errors detected by lintian
Author: Rolf Leggewie

Based on work by Philipp Matthias Hahn

--- a/doc/roger.1.in
+++ b/doc/roger.1.in
@@ -8,7 +8,7 @@ roger \- this is the graphical user interface of Roger Router.
 .B roger [OPTIONS...]
 
 .SH DESCRIPTION
-\fBRoger Router\fR is a clean solution for controlling the FRITZ!Box or compatible router with Linux. It offers a rich feature list, including caller monitor, fax and softphone support.
+\fBroger\fR offers a clean solution for controlling the FRITZ!Box or compatible router with Linux. It offers a rich feature list, including caller monitor, fax and softphone support.
 
 .SH OPTIONS
 .TP
--- a/doc/roger_cli.1.in
+++ b/doc/roger_cli.1.in
@@ -1,4 +1,4 @@
-.\" Copyright (c) 2009-2014, Jan-Michael Brummer <jan.brummer@tabos.org>
+.\" Copyright (c) 2009-2014,\ Jan-Michael Brummer <jan.brummer@tabos.org>
 .TH roger_cli 1 "" "Roger Router CLI \- Version @PACKAGE_VERSION@" "Roger Router CLI - User Manual"
 
 .SH NAME
@@ -8,35 +8,37 @@ roger_cli \- this is the command line interface of Roger Router.
 .B roger_cli [OPTION...]
 
 .SH DESCRIPTION
-\fBRoger Router\fR is a clean solution for controlling the FRITZ!Box or compatible router with Linux. It offers a rich feature list, including caller monitor, fax and softphone support.
+\fBroger_cli\fR offers a clean solution for controlling the FRITZ!Box or compatible router from the command line with Linux. It can be used to print the current FRITZ!Box journal to the screen,\ initiate calls or send facsimiles.
 
 .SH OPTIONS
+The program follows the usual GNU command line syntax,\ with long options starting with two dashes (`-').
+A summary of options is included below.
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+Show help options.
 .TP
 \fB\-c\fR, \fB\-\-call\fR
 Initiate an outgoing call by softphone. Requires \fB\-n\fR \fInumber\fR to be given as well.
 .TP
-\fB\-d\fR, \fB\-\-debug\fR
-Enable debug messages.
-.TP
-\fB\-f\fR \fIfile\fR, \fB\-\-file\fR=\fIfile\fR
-File name of a PDF or PostScript \fIfile\fR containing the fax to send.
-.TP
-\fB\-h\fR, \fB\-\-help\fR
-Show help options.
+\fB\-s\fR, \fB\-\-sendfax\fR
+Initiate sending of a fax. Requires \fB\-f\fR \fIfile\fR (in PS or PDF format) and \fB\-n\fR \fInumber\fR to be given as well.
 .TP
 \fB\-j\fR, \fB\-\-journal\fR
 Show journal of incoming and outgoing calls on the screen.
 .TP
-\fB\-s\fR, \fB\-\-sendfax\fR
-Initiate sending of a fax. Requires \fB\-f\fR \fIfile\fR and \fB\-n\fR \fInumber\fR to be given as well.
+\fB\-d\fR, \fB\-\-debug\fR
+Enable debug messages.
 .TP
 \fB\-n\fR \fInumber\fR, \fB\-\-number\fR=\fInumber\fR
 Remote telephone or fax \fInumber\fR.
+.TP
+\fB\-f\fR \fIfile\fR, \fB\-\-file\fR=\fIfile\fR
+File name of a PDF or PostScript \fIfile\fR containing the fax to send.
 
 .SH NOTES
 .IP "\fIFaxing\fR" 4
 .RS
-If you want to use the capi fax function into Roger Router, you must enable capi-over-tcp within your FRITZ!Box with #96*3* and the user must be a member of the group fax.
+If you want to use the capi fax function with Roger Router, you must enable capi-over-tcp within your FRITZ!Box with #96*3* and the user must be a member of the group fax.
 .RE
 .IP "\fINotifications\fR" 4
 .RS
@@ -69,4 +71,8 @@ Jan-Michael Brummer (developer)  <\fIjan.brummer@tabos.org\fR>
 .IP "This manpage was written by:" 4
 .RS
 Jan-Michael Brummer <\fIjan.brummer@tabos.org\fR>
+.br
+Philipp Matthias Hahn <\fIpmhahn@debian.org\fR>
+.br
+Rolf Leggewie <\fIfoss@rolf.leggewie.biz\fR>
 .RE
