Subject: Fix manpage errors detected by lintian
Author: Rolf Leggewie

Based on work by Philipp Matthias Hahn

--- a/doc/roger.1.in
+++ b/doc/roger.1.in
@@ -6,8 +6,7 @@
 .B roger
 
 .SH DESCRIPTION
-.PP
-\fBRoger Router\fP is a clean solution for controlling the FRITZ!Box or compatible
+\fBroger\fP is a clean solution for controlling the FRITZ!Box or compatible
 router with Linux. It offers a rich feature list, including caller monitor,
 fax and softphone support.
 
--- a/doc/roger_cli.1.in
+++ b/doc/roger_cli.1.in
@@ -1,32 +1,40 @@
-.\" Copyright (c) 2009-2014, Jan-Michael Brummer <jan.brummer@tabos.org>
+.\" Copyright (c) 2009-2014,\ Jan-Michael Brummer <jan.brummer@tabos.org>
 .TH roger_cli 1 "" "Roger Router CLI Version @PACKAGE_VERSION@"
 .SH NAME
-roger \- Command Line Router Manager
+roger_cli \- Command Line Router Manager
 .SH SYNOPSIS
 .B roger_cli
 
 .SH DESCRIPTION
-\fBroger_cli\fP can print the current FRITZ!Box journal to the screen and send fax files.
+\fBroger_cli\fP is the Roger Router command line client for managing the FRITZ!Box.
+It can be used to print the current FRITZ!Box journal to the screen,\ initiate calls or send facsimiles.
 
 .SH OPTIONS
+The program follows the usual GNU command line syntax,\ with long options starting with two dashes (`-').
+A summary of options is included below.
 .TP
-.BR \-c
-Call a number with the softphone. Needs target number, see -n.
+.B \-h,\ \-\-help
+Show summary of options.
 .TP
-.BR \-s
-Send fax file. Needs fax file name (PS/PDF) -f and target number -n.
+.B \-d,\ \-\-debug
+Enable debug messages.
 .TP
-.BR \-d
-Enable debug output.
+.B \-j,\ \-\-journal
+Prints journal of incoming and outgoing calls.
 .TP
-.BR \-j
-Show journal.
+.B \-s,\ \-\-sendfax
+Initiate sending of a facsimile.
+Requires \-f \fIfile\fP (in PS or PF format) and \-n \fInumber\fP to be given as well.
 .TP
-.BR \-f
-File name of fax to transfer.
+.B \-c,\ \-\-call
+Initiate an outgoing call by softphone.
+Requires \-n \fInumber\fP to be given as well.
 .TP
-.BR \-n
-Target number.
+.B \-f \fIfile\fP,\ \-\-file=\fIfile\fP
+File name of a PDF or PostScript \fIfile\fP containing the facsimile to send.
+.TP
+.B \-n \fInumber\fP,\ \-\-number=\fInumber\fP
+Remote telephone or facsimile \fInumber\fP.
 
 .SH SEE ALSO
 \fIhttp://www.tabos.org/roger/\fP
@@ -40,4 +48,4 @@
  Jan-Michael Brummer (developer)
 .br
 
-This manpage was written by Jan-Michael Brummer <\fIjan.brummer@tabos.org\fP>.
\ No newline at end of file
+This manpage was written by Jan-Michael Brummer <\fIjan.brummer@tabos.org\fP> with additions by Philipp Matthias Hahn <\fIpmhahn@debian.org\fP> and Rolf Leggewie <\fIfoss@rolf.leggewie.biz\fP>.
