Subject: Force building CUPS backend without CUPS being installed.
Author: Philipp Matthias Hahn

--- a/configure.ac
+++ b/configure.ac
@@ -221,13 +221,9 @@
 		[ cupsbackenddir="${withval}" ],
 		[ cupsbackenddir=`cups-config --serverbin`/backend ]
 	)
-	if test -d "$cupsbackenddir"; then
 		AC_SUBST([cupsbackenddir])
 		AC_MSG_RESULT([$cupsbackenddir])
 		AC_DEFINE([HAVE_CUPS_BACKEND], 1, [Use new CUPS backend])
-	else
-		AC_MSG_FAILURE([Cups backend directory not found!])
-	fi
 fi
 
 # Check for unistd.h
